plugins/field-dependent-dropdown/src/field_dependent_dropdown.ts

```diff
@@ -0,0 +1,354 @@
+/**
+ * @license
+ * Copyright 2023 Google LLC
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+/**
+ * @fileoverview A tool for making a dropdown field's options depend on the
+ * value of another field.
+ */
+
+import * as Blockly from 'blockly/core';
+import {Block} from 'blockly/core/block';
```

Comment:
Importing specific modules from outside of core is not currently supported. Instead, in a plugin, each of these should be referenced as `Blockly.Block` etc.

It's interesting to me that this seems to work for you, but this isn't something we currently recommend that people do, and it may change in the future depending on Christopher's work. @cpcallen any thoughts on this?

Reply:
Done.